cli/browser-demo/index.html

```diff
@@ -16,10 +15,13 @@
 	log('Loading WASM');
 	await init('./pkg/polkadot_cli_bg.wasm');
 	log('Successfully loaded WASM');
+	log('Fetching chain spec');
+	const chain_spec_response = await fetch("https://raw.githubusercontent.com/paritytech/polkadot/master/service/res/westend.json");
```

Comment:
As mentioned in the browser-light-client chat: I don't think that can work, as the format of the database and the network message are directly derived from the runtime's code.

Reply:
It's also ideologically questionable to download the chainspecs from something that Parity owns, as the entire point of the blockchain client is to be isolated.
If GitHub goes down, the node doesn't work.

Reply:
The node devs have to be trusted in any case, but I agree that we shouldn't be downloading from GitHub if avoidable. Still, I view in-browser light client as a prototype at the moment, where we can make some allowances as nobody uses it yet :)